### PR TITLE
CP-27559: Refactor SSHCommand

### PR DESCRIPTION
**Pull request by DeliZhangX**
Signed-off-by: Deli Zhang <Deli.Zhang@citrix.com>

### CODE REVIEW

File: autocertkit/ssh.py [hunk 1]
```diff
@@ -179,71 +178,67 @@ def __init__(self,
...
-
-            If "string" is used then a failure results in an exception.
-
-        """
-
-        if self.toreply:
-            if retval == "string":
-                raise Exception(self.reply)
-            return self.reply
+    def read_file(self, inf, outf=None, label="stdout"):
```
Comment (acefei) [inline autocertkit/ssh.py]: add doc string to illustrate how to use the function.

Comment (DeliZhangX) [reply autocertkit/ssh.py]: Needless I think, there is doc string for the original interface, see below 'read' function.
In my opinion, we should keep the "Style" during refactor, unless we modify all.

File: autocertkit/ssh.py [hunk 2]
```diff
@@ -164,9 +164,8 @@ def __init__(self,
                  timeout=300,
                  password=None,
                  nowarn=False,
-                 newlineok=False,
```
Comment (Zhengchai) [inline autocertkit/ssh.py]: why we remove this line?

Comment (DeliZhangX) [reply autocertkit/ssh.py]: After refactoring, the stderr is able to return, so needless to check how many newline in command, indicates use can run any command, this lib don't care what they are, even incorrect commands.

**Pull request merged**